tests/unit/CourtOrderPoa.spec.ts

```diff
@@ -98,26 +112,29 @@ describe('Court Order and Plan of Arrangement component', () => {
     // Verify checkbox is selected
     expect(wrapper.vm.$data.planOfArrangement).toBe(true)
 
-    // Input text into text-field
-    const input = wrapper.find('#court-order-number-input')
-    input.setValue('Test')
-    await Vue.nextTick()
-
-    // Prompt validates through prop
-    wrapper.setProps({ validate: true })
-    await Vue.nextTick()
-
+    // Verify it continues invalid
     expect(wrapper.find('#court-num-form').text()).toContain('Court order number is invalid')
-    expect(wrapper.emitted('emitValid').pop()[0]).toEqual(false)
+    expect(wrapper.emitted().emitValid.length).toBe(1)
```

Comment:
Checking the number of events emitted is good, but you should definitely verify that the last event has the correct value.

Reply:
It is checked on line 105.

Reply:
Ok, I see now (and your code comment - thanks) You can always check it again. Up to you.